docs/game-provider.md

```diff
@@ -0,0 +1,18 @@
+# Game provider
+
+Game providers are specified through 2 layers
+
+### Hardcoded logic
+
+Hardcoded game providers MUST be stored in separate module, all well-documented and independently versioned.
+
+### Game provider spec
+
+Game providers are generated statically at the beginning of simulation and do not hold internal state. Spec file should be a TOML in dedicated subdiractory of the model.
```

Comment:
Do you aim for two separate code compilation/execution steps? First one to read configs and build required binary, second one to run the game itself.

Reply:
Yes, these should be separated. We'll have clearly defined holography here then, with configs as boundary conditions and run as internal space.